scss/forms/_form-check.scss

```diff
@@ -173,3 +173,12 @@
     }
   }
 }
+
+@if $enable-dark-mode {
+  @include color-mode(dark) {
+    // Use `:not()` to keep from having to override subsequent embedded SVGs
+    .form-switch .form-check-input:not(:checked):not(:focus) {
+      background-image: escape-svg($form-switch-bg-image-dark);
```

Comment:
Couldn't we do the same thing as in `forms/_form-select.scss` we a CSS var instead of having this `background-image` rule?

Reply:
This comment is handled by https://github.com/twbs/bootstrap/pull/35857/commits/cd01c4b60b6569c98427017d2a1bbe623621a667. However, we don't see anymore the ticks in light/dark modes.
![Screenshot 2022-11-19 at 10 15 28](https://user-images.githubusercontent.com/17381666/202843911-605e7662-1674-45d9-b2ce-4f73fb5b785e.png)
![Screenshot 2022-11-19 at 10 15 23](https://user-images.githubusercontent.com/17381666/202843913-aa1008a1-fa1e-4226-b190-3541126bc92f.png)
![Screenshot 2022-11-19 at 10 15 19](https://user-images.githubusercontent.com/17381666/202843914-a69af358-123a-4095-9ecc-c81c165a0c03.png)
![Screenshot 2022-11-19 at 10 15 13](https://user-images.githubusercontent.com/17381666/202843915-afc0c9f7-1f3c-478c-a4bb-58ce28aab4b9.png)
![Screenshot 2022-11-19 at 10 15 09](https://user-images.githubusercontent.com/17381666/202843917-b5c4ad77-c0fa-4871-ad1d-12b5407e95b1.png)



(Switches are OK)